data/json/items/armor/jewelry.json

```
@@ -1123,6 +1151,17 @@
     "use_action": [ "MEDITATE" ],
     "flags": [ "NO_WEAR_EFFECT" ]
   },
+  {
+    "id": "wooden_hair_beads",
+    "copy-from": "garnet_silver_tiara",
+    "type": "ARMOR",
+    "name": { "str": "wooden hair beads" },
```

```suggestion
    "name": { "str": "wooden hair beads", "str_pl": "sets of wooden hair beads" },
```
If you don't like "sets of" and can't think of anything else you can make it `"name": { "str_sp": "wooden hair beads" }` instead